Core.Collectors/Web/HttpClientWrapper.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using Microsoft.ApplicationInsights;
```

Comment:
No other changes here, was the `using` directive missing before? Otherwise, revert this file.